test/Sign.SignatureProviders.TrustedSigning.Test/TrustedSigningServiceProviderTests.cs

```diff
@@ -2,135 +2,63 @@
+                     Mock.Of<CertificateProfileClient>(),
+                     "account",
+                     "profile",
+                     sp.GetRequiredService<ILogger<TrustedSigningService>>()
+                     );
+            });
             serviceProvider = services.BuildServiceProvider();
         }
 
-        [Fact]
-        public void Constructor_WhenTokenCredentialIsNull_Throws()
-        {
-            ArgumentNullException exception = Assert.Throws<ArgumentNullException>(
-                () => new TrustedSigningServiceProvider(tokenCredential: null!, EndpointUrl, AccountName, CertificateProfileName));
-
-            Assert.Equal("tokenCredential", exception.ParamName);
-        }
-
-        [Fact]
-        public void Constructor_WhenEndpointUrlIsNull_Throws()
-        {
-            ArgumentNullException exception = Assert.Throws<ArgumentNullException>(
-                () => new TrustedSigningServiceProvider(TokenCredential, endpointUrl: null!, AccountName, CertificateProfileName));
-
-            Assert.Equal("endpointUrl", exception.ParamName);
-        }
-
-        [Fact]
-        public void Constructor_WhenAccountNameIsNull_Throws()
-        {
-            ArgumentNullException exception = Assert.Throws<ArgumentNullException>(
-                () => new TrustedSigningServiceProvider(TokenCredential, EndpointUrl, accountName: null!, CertificateProfileName));
-
-            Assert.Equal("accountName", exception.ParamName);
-        }
-
-        [Fact]
-        public void Constructor_WhenAccountNameIsEmpty_Throws()
-        {
-            ArgumentException exception = Assert.Throws<ArgumentException>(
-                () => new TrustedSigningServiceProvider(TokenCredential, EndpointUrl, accountName: string.Empty, CertificateProfileName));
-
-            Assert.Equal("accountName", exception.ParamName);
-        }
-
-        [Fact]
-        public void Constructor_WhenCertificateProfileNameIsNull_Throws()
-        {
-            ArgumentNullException exception = Assert.Throws<ArgumentNullException>(
-                () => new TrustedSigningServiceProvider(TokenCredential, EndpointUrl, AccountName, certificateProfileName: null!));
-
-            Assert.Equal("certificateProfileName", exception.ParamName);
-        }
-
-        [Fact]
-        public void Constructor_WhenCertificateProfileNameIsEmpty_Throws()
-        {
-            ArgumentException exception = Assert.Throws<ArgumentException>(
-                () => new TrustedSigningServiceProvider(TokenCredential, EndpointUrl, AccountName, certificateProfileName: string.Empty));
-
-            Assert.Equal("certificateProfileName", exception.ParamName);
-        }
-
         [Fact]
         public void GetSignatureAlgorithmProvider_WhenServiceProviderIsNull_Throws()
         {
-            TrustedSigningServiceProvider provider = new(TokenCredential, EndpointUrl, AccountName, CertificateProfileName);
+            TrustedSigningServiceProvider provider = new();
```

Comment:
Since all tests do this, you could promote this to a field and make the bodies of the tests more succinct.

```C#
private readonly TrustedSigningServiceProvider _provider = new();
```